### ISSUE TITLE
Unused equalTo() when overriding equal()

### ISSUE THREAD

**Issue by bcalmac** [immutables/immutables#266]
When I override equals() in my abstract value class, the generated class contains an unused equalTo():

```java
  private boolean equalTo(Point2D another) {
    return super.equals(another);
  }
```

It doesn't harm anything, but when equals() is omitted from the generated class, equalTo() should follow.

**Comment by elucash** [comment]
Hm, I thought we had a check for that, but good to know. Thank you for spotting this!

**Comment by elucash** [comment]
Released in 2.1.12, thank you!
